### ISSUE TITLE
False positive when 'using static' with a drop-in-replacement

### ISSUE THREAD

**Issue by LukasGelke** [inyutin-maxim/System.IO.Abstractions.Analyzers#74]
As I was porting several classes to using Abstractions, i created a class like this
```cs
namespace Mocking
{
  public static class FileSystemProxy
  {
    internal static IFileSystem FileSystem { get; set; } = new FileSystem();

    internal static IFile File => FileSystem.File;

    internal static IDirectory Directory => FileSystem.Directory;
  }
}
```
and replaced `using System.IO;` with `using static Mocking.FileSystemProxy;`.
In this way, code changes per file can be minimized.
This however results in the following problem: even though `File` is now an `IFileSystem.File` from Abstractions, the analyzer still complains, and wants it to be replaced.

![grafik](https://user-images.githubusercontent.com/71026023/97280494-a200d500-183c-11eb-8499-df44ec937cce.png)

**My Suggestion**
Fix analyzers so that they don't only check by name/string. In this case, one can obtain a symbol and see, that it is in fact from Abstractions. (System.IO.File returns an empty SymbolInfo in my test)
![grafik](https://user-images.githubusercontent.com/71026023/97286313-a2509e80-1843-11eb-9101-e7bc21e9562e.png)

**Comment by inyutin-maxim** [comment]
@LukasGelke Thanks for the suggestions. 
I fix this problem later when I find a time)
You can try to resolve this problem and send me PR.
